v22.1/upsert.md

```diff
@@ -237,6 +239,77 @@ In such a case, you would need to use the [`INSERT ON CONFLICT`](insert.html) st
 +---+---+
 ~~~
 
+### Upsert a proper subset of columns
+
+~~~ sql
+> CREATE TABLE accounts (
+    id INT PRIMARY KEY,
+    name STRING,
+    balance DECIMAL(10, 2) DEFAULT 0
+);
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> INSERT INTO accounts (id, name, balance) VALUES
+    (1, 'a1', 10000.5),
+    (2, 'b1', 20000.75),
+    (3, 'c1',  6325.2);
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT * FROM accounts;
+~~~
+
+~~~
++----+------+----------+
+| id | name | balance  |
++----+------+----------+
+|  1 |   a1 | 10000.50 |
+|  2 |   b1 | 20000.75 |
+|  3 |   c1 |  6325.20 |
++----+------+----------+
+~~~
+
+Upserting a proper subset of columns without specifying the column names will write the default values of the unspecified columns when there is a conflict on the primary key. Notice how the account with `id` of `1` has a balance of `0` (the column's default value) after the `UPSERT`:
```

Comment:
@mgartner What do you mean "when there is a conflict on the primary key"?  I thought the point is simply that the target columns are missing.

Reply:
I was trying to convey that this slight behavioral difference has no effect when the `UPSERT` inserts a row, which happens when there is no conflict on the primary key. The default column value will always be used. Perhaps this is obvious — there's no other value that could be used.

Example:

```
defaultdb> CREATE TABLE accounts (
    id INT PRIMARY KEY,
    name STRING,
    balance DECIMAL(10, 2) DEFAULT 0
);
CREATE TABLE

defaultdb> SELECT * FROM accounts;
  id | name | balance
-----+------+----------
(0 rows)

defaultdb> UPSERT INTO accounts VALUES (1, 'a1');
INSERT 1

defaultdb> UPSERT INTO accounts (id, name) VALUES (2, 'b1');
INSERT 1

defaultdb> SELECT * FROM accounts;
  id | name | balance
-----+------+----------
   1 | a1   |    0.00
   2 | b1   |    0.00
(2 rows)
```